proposals-CG-WG.md

```diff
@@ -38,42 +38,45 @@ The purpose of Stage 1 proposals is to explore the Solution Space and pick a pre
 
 # Stage 2: Formalization
 
-The purpose of Stage 2 Proposals is to formally specify the best (and seemingly workable) alternative that was identified in the prior step: handle corner cases, integrate with other parts, reconcile with other proposals, and resolve the concerns identified at the entrance. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.
+The goal of Stage 2 is to refine the preferred solution into a detailed, cohesive proposal that addresses known issues and integrates feedback from stakeholders. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.  This stage focuses on preparing the proposal for Working Group review and creating a complete, formal draft.
 
   * What's needed from champions?
-    * [ ] An [explainer](https://tag.w3.org/explainers/)
-    * [ ] Alternatives and trade-offs considered
-    * [ ] A specific preferred proposal, e.g., code samples, examples, or a spec PRs (if the formalism is needed to understand the proposal)
-    * [ ] Browser and developer implementation experience (e.g., a prototype, dev trials, origin trials, etc) 
-    * [ ] Confidence of developer demand and fitness for purpose (e.g. a developer that needs this proposal)
+    * [ ] An [explainer](https://tag.w3.org/explainers/).
+    * [ ] Documentation of alternatives and trade-offs considered.
+    * [ ] Optionally, a draft specification text (or detailed examples/code samples if needed for clarity).
+    * [ ] Early implementation experience, including prototypes or trials.
+    * [ ] Evidence of stakeholder (e.g. web developers) demand and use-case alignment.
+
   * What's asked of the **Working Group**?
-    * [ ] Working Group consensus to adopt the proposal as the basis for their work as a [Working Draft](https://www.w3.org/policies/process/#RecsWD).
-    * [ ] Working Group identification of the list of (seemingly resolvable) issues that have to be addressed before [Stage 3](#stage-3).
-    * [ ] An [Editor's Draft](https://www.w3.org/policies/process/#editors-draft) can be used to get PRs merged between [Working Draft](https://www.w3.org/policies/process/#RecsWD) revisions.
+    * [ ] Working Group consensus to adopt the proposal as the basis for their work.
+    * [ ] A clear list of blocking issues to be addressed before advancing to Stage 3.
+    * [ ] A completed Working Draft for further iteration.
```

Comment:
Link to `Working Draft` seemed useful, why was it removed?

Reply:
Ooops.